src/arrayReverse.test.js

```diff
@@ -1,20 +1,35 @@
 'use strict';
 
-describe(`Function 'arrayReverse':`, () => {
+describe('arrayReverse', () => {
   const arrayReverse = require('./arrayReverse');
 
-  it(`should be declared`, () => {
+  it('should be declared', () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it('should reverse the order of the words in the array', () => {
+    const words = ['hello', 'world'];
+    const reversed = arrayReverse(words);
 
+    expect(reversed).toEqual(['dlrow', 'olleh']);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it(`should return an empty 
+string if original array consists of an empty string`, () => {
```

Comment:
It should return not an empty string, but an array with only one single string. Test description is a bit confusing